helm/templates/rbac/role.yaml

```diff
@@ -0,0 +1,67 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: manager-role
```

Comment:
The role name used here is too generic. We could use something like: name: {{ include "etcd-operator.name" . }}-role. Same comment applies to all other places to help identify that the resource belongs to etcd-operator